backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Core/ClientInterfaces/IResourceRegistryClient.cs

```diff
@@ -20,6 +21,12 @@ public interface IResourceRegistryClient
         /// </summary>
         /// <returns>The resource list if exists</returns>
         Task<List<ServiceResource>> GetResources();
+        
+        /// <summary>
+        ///     Integration point for retrieving the list of resources for the given <param name="resourceType"></param> 
+        /// </summary>
+        /// <returns>The resource list if exists</returns>
+        Task<List<ServiceResource>> GetResources(ResourceType resourceType);
```

Comment:
It works now. But since we can't filter on every resourceType it's a little bit useless to automate that endpoint for us so I removed the whole method that sends in ResourceType.